pkg/hive/hive.go

```diff
@@ -213,7 +213,8 @@ func (h *Hive) Run() error {
 func (h *Hive) waitForSignalOrShutdown() error {
 	signals := make(chan os.Signal, 1)
 	defer signal.Stop(signals)
-	signal.Notify(signals, os.Interrupt, unix.SIGINT, unix.SIGTERM)
+	// equivalent to unix.SIGINT and unix.SIGTERM, but works on cross compilation for Windows
+	signal.Notify(signals, os.Interrupt, syscall.Signal(0x2), syscall.Signal(0xf))
```

Comment:
The `os.Interrupt` is the same as `unix.SIGINT` so `SIGINT` can be removed. Let's not use magic constants here, but use instead `syscall.SIGTERM`. It's also defined for Windows: https://github.com/golang/go/blob/master/src/syscall/types_windows.go#LL51C9-L51C9, but likely does nothing.

Reply:
Thanks for your review, updated as per the suggestion :heavy_check_mark: